kornia/contrib/object_detection.py

```diff
@@ -22,13 +22,14 @@ def __init__(self, size: int | tuple[int, int], interpolation_mode: str = "bilin
         """
         super().__init__()
         self.size = size
+        self.resizer = Resize((self.size, self.size))
```

Comment:
```suggestion
        self.resizer = Resize(self.size)
```

Reply:
@edgarriba  If size is an int, then smaller edge of the image will be matched to size right?

Reply:
yes

Reply:
i believe that this logic is already handled by the Resize class

Reply:
@johnnv1 I tried using `side` parameter but only one  dimension of the image is resized. I tried all the possible values for `side` parameter

Reply:
These were the test-cases i tried for `side` parameter. And only one dimension is getting resized when `size` is an integer

```bash

>>> from kornia.geometry.transform import Resize
>>> res = Resize(400, side="long", interpolation="bilinear")
>>> img = torch.randn(3, 600, 300, device="cpu", dtype=torch.float32)
>>> img_resized = res(img)
>>> img_resized.shape 
torch.Size([3, 400, 200])

//  side="vert"

>>> res = Resize(400, side="vert", interpolation="bilinear")
>>> img_resized.shape
torch.Size([3, 400, 200])

// side="horz"

>>> res = Resize(400, side="horz", interpolation="bilinear")
>>> img_resized = res(img)
>>> img_resized.shape
torch.Size([3, 800, 400])

// side="short"

>>> res = Resize(400, side="short", interpolation="bilinear")
>>> img_resized = res(img)
>>> img_resized.shape
torch.Size([3, 800, 400])



```

Reply:
These examples look right to me 

- when `vert` the vertical side (height -- dim 1) is resized to 400, and the width keeps the aspect ratio and goes to 200
- when `short` the shortest side (width on your ex image) is resized to 400, and the height goes to 800 to keep the aspect ratio

if you want to resize both sizes for the exact size you need to resize then pad the image (not causing any distortion keeping the aspect ratio)

Reply:
@johnnv1 So If i gave  an interger for `size` in `Resize`, Is there any parameter for `Resize` class  to resize both the dimention of the image

> i believe that this logic is already handled by the Resize class

Reply:
i belive not, this case we should pass a tuple

Reply:
@johnnv1 Updated the changes